src/libexpr/primops/fetchTree.cc

```diff
@@ -125,7 +125,7 @@ static void fetchTree(
             if (attr.name == state.sType) continue;
             state.forceValue(*attr.value, attr.pos);
             if (attr.value->type() == nPath || attr.value->type() == nString) {
-                auto s = state.coerceToString(attr.pos, *attr.value, context, false, false, "").toOwned();
+                auto s = state.coerceToString(attr.pos, *attr.value, context, "", false, false).toOwned();
```

Comment:
@tomberek: here it was missing one bool, and the empty string was passed in lieu of the third bool.
Overall, it was silly to provide a default value for that error context, which I wanted to be required. So it makes sense to move it before optional arguments.